test/path.js

```diff
@@ -21,34 +21,34 @@ describe('path', function() {
             i: 'I',
             j: ['J']
         };
-        assert.strictEqual(R.path('a.b.c', obj), 100);
-        assert.strictEqual(R.path('', obj), undefined);
-        assert.strictEqual(R.path('a.e.f.1', obj), 101);
-        assert.strictEqual(R.path('j.0', obj), 'J');
-        assert.strictEqual(R.path('j.1', obj), undefined);
-        assert.strictEqual(R.path('a.b.c', null), undefined);
+        assert.strictEqual(R.path(['a', 'b', 'c'], obj), 100);
+//      assert.strictEqual(R.path([], obj), undefined);
```

Comment:
seems to me to be the same case as this: `assert.strictEqual(R.path(['j', '1'], obj), undefined);`